packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDElements/PhotoCaptureHUDElements.model.ts

```diff
@@ -0,0 +1,69 @@
+import { CaptureAppConfig, Image, PixelDimensions, Sight } from '@monkvision/types';
+import { AddDamageHandle, PhotoCaptureMode, TutorialSteps } from '../../hooks';
+
+/**
+ * Props of the PhotoCaptureHUDElements component.
+ */
+export interface PhotoCaptureHUDElementsProps
+  extends Pick<CaptureAppConfig, 'enableSightGuidelines' | 'sightGuidelines' | 'addDamage'> {
+  /**
+   * The currently selected sight in the PhotoCapture component : the sight that the user needs to capture.
+   */
+  selectedSight: Sight;
+  /**
+   * The list of sights provided to the PhotoCapture component.
+   */
+  sights: Sight[];
+  /**
+   * Array containing the list of sights that the user has already captured.
+   */
+  sightsTaken: Sight[];
+  /**
+   * The current mode of the component.
+   */
+  mode: PhotoCaptureMode;
+  /**
+   * The current tutorial step in PhotoCapture component.
+   */
+  tutorialStep: TutorialSteps | null;
+  /**
+   * Callback to be called when the user clicks on the "Add Damage" button.
+   */
+  onAddDamage: AddDamageHandle['handleAddDamage'];
+  /**
+   * Callback to be called when the user selects the parts to take a picture of.
+   */
+  onAddDamageParts: AddDamageHandle['handleAddParts'];
+  /**
+   * The current state of the Add Damage mode of part selection.
+   */
+  addDamagePartSelectState: 'part-select' | 'image-capture';
```

Comment:
Use an enum instead of magic strings for 'part-select' and 'image-capture'

Reply:
this change had been removed.